blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -70,6 +72,11 @@ gigabytes of heap for both sbt and bloop (the shell, not the nailgun
 integration). When benchmarking sbt, we have made sure that dependency
 resolution and formatting are not measured.
 
+To put those numbers into context, the speedup observed in `apache/spark` is
+equal to the compilation speedup that Jason Zaugg introduced in the 2.12.x
+series. That means that when Spark developers migrate to 2.12 and use `bloop`,
+they will experience a compilation speedup close to 80%.
```

Comment:
I think you mean 20% speedup?

Reply:
or if you really mean 80%, it's unclear how that figure is arrived it, so it needs a change regardless

Reply:
When you say "speedup", you mean that you do 80% better than previously, right?

A speedup of 2 is a speedup of 200%, which is that you do 2 times better than before.

Reply:
@SethTisue I've addressed your feedback